src/hashesorg.py

```diff
@@ -0,0 +1,26 @@
+def HashesOrg(hashed, key):
+    try:
+        output = requests.get(f"https://hashes.org/api.php?key={key}&query={hashed}", timeout=5).text

```

Comment:
5 second timeout, and it's not multi-threaded? This would slow the program down a lot, I would do:
1. Loop over decryptors
2. Multi-thread call `class.decrypt()`

This means our program doesn't wait 60+ seconds to do all the work :))